WalletWasabi/WabiSabi/Models/InputsRegistrationRequest.cs

```diff
@@ -0,0 +1,24 @@
+using NBitcoin;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using WalletWasabi.WabiSabi.Crypto;
+
+namespace WalletWasabi.WabiSabi.Models
+{
+	public class InputsRegistrationRequest
+	{
+		public InputsRegistrationRequest(Guid roundId, IEnumerable<InputRoundSignaturePair> inputRoundSignaturePairs, RegistrationRequestMessage zeroCredentialRequests)
+		{
+			RoundId = roundId;
+			InputRoundSignaturePairs = inputRoundSignaturePairs;
+			ZeroCredentialRequests = zeroCredentialRequests;
+		}
+
+		public Guid RoundId { get; }
+		public IEnumerable<InputRoundSignaturePair> InputRoundSignaturePairs { get; }
+		public RegistrationRequestMessage ZeroCredentialRequests { get; }
```

Comment:
Idk what changed but in theory these can be zero or non-zero credentials. I would rename it as `CredentialRequests`